File: typescript/src/n0242_valid_anagram.test.ts
import { expect } from "jsr:@std/expect";
import { isAnagram } from "./n0242_valid_anagram.ts";

Deno.test("isAnagram", () => {
  expect(isAnagram("anagram", "nagaram")).toBe(true);
  expect(isAnagram("rat", "car")).toBe(false);
  expect(isAnagram("listen", "silent")).toBe(true);
  expect(isAnagram("a", "ab")).toBe(false);
});

File: rust/src/n0242_valid_anagram.test.rs
use crate::n0242_valid_anagram::*;

#[test]
fn test_is_anagram() {
    assert!(is_anagram("anagram", "nagaram"));
    assert!(!is_anagram("rat", "car"));
    assert!(is_anagram("listen", "silent"));
    assert!(!is_anagram("a", "ab"));
}
